merk/src/proofs/encoding.rs

```diff
@@ -36,6 +43,7 @@ impl Encode for Op {
         Ok(match self {
             Op::Push(Node::Hash(_)) => 1 + HASH_LENGTH,
             Op::Push(Node::KVHash(_)) => 1 + HASH_LENGTH,
+            Op::Push(Node::KVDigest(key, _)) => 2 + key.len() + HASH_LENGTH,
```

Comment:
Just curious, why 2?

Reply:
0x04 and key.len() as u8 take 1 byte each (making two bytes)
rest is key and value_hash which  are accounted for with the rest of the equation (key.len() and HASH_LENGTH)

Reply:
Got it